monitoring/prometheus/metric.go

```diff
@@ -2,24 +2,20 @@ package prometheus
 
 import (
 	"context"
+	pluginpb "github.com/dsrvlabs/vatz-proto/plugin/v1"
 	"github.com/dsrvlabs/vatz/manager/config"
-	"github.com/dsrvlabs/vatz/utils"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
-func (c *prometheusManager) getPluginUp(plugins []config.Plugin, hostName string) (
-	pluginUp map[int]*prometheusValue,
-) {
-	gClients := utils.GetClients(plugins)
+func (c *prometheusManager) getPluginUp(plugins []config.Plugin, hostName string, grpcClients []pluginpb.PluginClient) (pluginUp map[int]*prometheusValue) {
 	pluginUp = make(map[int]*prometheusValue)
-
 	for idx, plugin := range plugins {
 		pluginUp[plugin.Port] = &prometheusValue{
 			Up:         1,
 			PluginName: plugin.Name,
 			HostName:   hostName,
 		}
-		verify, err := gClients[idx].Verify(context.Background(), new(emptypb.Empty))
+		verify, err := grpcClients[idx].Verify(context.Background(), new(emptypb.Empty))
```

Comment:
do you have any specific reason to change this?
any bugs on previous codes?

Reply:
No, there wasn't any issue with it.
but I feel like it create same clients per purpose  
but I prefer to use same client which does exactly doing same things. 
Any concens?

Reply:
I just worried about the function's arguments. If we pass everything, you know, function's prototype could be messed up and sometimes it could be really hard to track the variables.

it depends on taste, forget it.

Reply:
> I just worried about the function's arguments. If we pass everything, you know, function's prototype could be messed up and sometimes it could be really hard to track the variables.
> 
> it depends on taste, forget it.

Agree
You are right. let me update little bit more and give me your feedback.